Using cpu device
C:\Users\rickv\anaconda3\envs\BlockB\lib\site-packages\stable_baselines3\common\vec_env\patch_gym.py:49: UserWarning: You provided an OpenAI Gym environment. We strongly recommend transitioning to Gymnasium environments. Stable-Baselines3 is automatically wrapping your environments in a compatibility layer, which could potentially cause issues.
  warnings.warn(
Wrapping the env with a `Monitor` wrapper
Wrapping the env in a DummyVecEnv.
Logging to runs/mzxiwkdn\runs/mzxiwkdn_0
[2KTraceback (most recent call last):[0m[38;5;237m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━[0m [32m841/100,000 [0m [ [33m0:00:00[0m < [36m0:01:02[0m , [31m1,623 it/s[0m ]
[2K  File "Training_locally.py", line 36, in <module>━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━[0m [32m841/100,000 [0m [ [33m0:00:00[0m < [36m0:01:02[0m , [31m1,623 it/s[0m ]
    model.learn(total_timesteps=time_steps, callback=wandb_callback, progress_bar=True, reset_num_timesteps=False,tb_log_name=f"runs/{run.id}")
[2K  File "C:\Users\rickv\anaconda3\envs\BlockB\lib\site-packages\stable_baselines3\ppo\ppo.py", line 311, in learn━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━[0m [32m841/100,000 [0m [ [33m0:00:00[0m < [36m0:01:02[0m , [31m1,623 it/s[0m ]
    return super().learn(
[2K  File "C:\Users\rickv\anaconda3\envs\BlockB\lib\site-packages\stable_baselines3\common\on_policy_algorithm.py", line 323, in learn━━━━━━━━━━━━━━━[0m [32m841/100,000 [0m [ [33m0:00:00[0m < [36m0:01:02[0m , [31m1,623 it/s[0m ]
    continue_training = self.collect_rollouts(self.env, callback, self.rollout_buffer, n_rollout_steps=self.n_steps)
[2K  File "C:\Users\rickv\anaconda3\envs\BlockB\lib\site-packages\stable_baselines3\common\on_policy_algorithm.py", line 224, in collect_rollouts━━━━[0m [32m841/100,000 [0m [ [33m0:00:00[0m < [36m0:01:02[0m , [31m1,623 it/s[0m ]
    if not callback.on_step():
[2K  File "C:\Users\rickv\anaconda3\envs\BlockB\lib\site-packages\stable_baselines3\common\callbacks.py", line 114, in on_step━━━━━━━━━━━━━━━━━━━━━━━[0m [32m841/100,000 [0m [ [33m0:00:00[0m < [36m0:01:02[0m , [31m1,623 it/s[0m ]
    return self._on_step()
[2K  File "C:\Users\rickv\anaconda3\envs\BlockB\lib\site-packages\stable_baselines3\common\callbacks.py", line 223, in _on_step━━━━━━━━━━━━━━━━━━━━━━[0m [32m841/100,000 [0m [ [33m0:00:00[0m < [36m0:01:02[0m , [31m1,623 it/s[0m ]
    continue_training = callback.on_step() and continue_training
[2K  File "C:\Users\rickv\anaconda3\envs\BlockB\lib\site-packages\stable_baselines3\common\callbacks.py", line 114, in on_step━━━━━━━━━━━━━━━━━━━━━━━[0m [32m841/100,000 [0m [ [33m0:00:00[0m < [36m0:01:02[0m , [31m1,623 it/s[0m ]
    return self._on_step()
[2K  File "C:\Users\rickv\anaconda3\envs\BlockB\lib\site-packages\wandb\integration\sb3\sb3.py", line 136, in _on_step━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━[0m [32m841/100,000 [0m [ [33m0:00:00[0m < [36m0:01:02[0m , [31m1,623 it/s[0m ]
    self.save_model()
[2K  File "C:\Users\rickv\anaconda3\envs\BlockB\lib\site-packages\wandb\integration\sb3\sb3.py", line 145, in save_model━━━━━━━━━━━━━━━━━━━━━━━━━━━━━[0m [32m841/100,000 [0m [ [33m0:00:00[0m < [36m0:01:02[0m , [31m1,623 it/s[0m ]
    wandb.save(self.path, base_path=self.model_save_path)
[2K  File "C:\Users\rickv\anaconda3\envs\BlockB\lib\site-packages\wandb\sdk\wandb_run.py", line 395, in wrapper_fn━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━[0m [32m841/100,000 [0m [ [33m0:00:00[0m < [36m0:01:02[0m , [31m1,623 it/s[0m ]
    return func(self, *args, **kwargs)
[2K  File "C:\Users\rickv\anaconda3\envs\BlockB\lib\site-packages\wandb\sdk\wandb_run.py", line 385, in wrapper━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━[0m [32m841/100,000 [0m [ [33m0:00:00[0m < [36m0:01:02[0m , [31m1,623 it/s[0m ]
    return func(self, *args, **kwargs)
[2K  File "C:\Users\rickv\anaconda3\envs\BlockB\lib\site-packages\wandb\sdk\wandb_run.py", line 2032, in save━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━[0m [32m841/100,000 [0m [ [33m0:00:00[0m < [36m0:01:02[0m , [31m1,623 it/s[0m ]
    return self._save(
[2K  File "C:\Users\rickv\anaconda3\envs\BlockB\lib\site-packages\wandb\sdk\wandb_run.py", line 2091, in _save━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━[0m [32m841/100,000 [0m [ [33m0:00:00[0m < [36m0:01:02[0m , [31m1,623 it/s[0m ]
    target_path.symlink_to(source_path)
[2K  File "C:\Users\rickv\anaconda3\envs\BlockB\lib\pathlib.py", line 1384, in symlink_to━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━[0m [32m841/100,000 [0m [ [33m0:00:00[0m < [36m0:01:02[0m , [31m1,623 it/s[0m ]
    self._accessor.symlink(target, self, target_is_directory)
[2KOSError: [WinError 1314] A required privilege is not held by the client: ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━[0m [32m841/100,000 [0m [ [33m0:00:00[0m < [36m0:01:02[0m , [31m1,623 it/s[0m ]
'C:\\Users\\rickv\\Documents\\GitHub\\2024-25b-fai2-adsai-RickVerburg232906\\Robotics\\models\\mzxiwkdn\\model.zip' ->
'C:\\Users\\rickv\\Documents\\GitHub\\2024-25b-fai2-adsai-RickVerburg232906\\Robotics\\wandb\\run-20241216_105732-mzxiwkdn\\files\\model.zip'
[35m   1%[0m [38;2;249;38;114m╸[0m[38;5;237m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━[0m [32m841/100,000 [0m [ [33m0:00:00[0m < [36m0:01:02[0m , [31m1,623 it/s[0m ]
